docs/projects-swagger.yaml

```diff
@@ -0,0 +1,416 @@
+  termsOfService: http://nva.unit.no/terms/
+  contact:
+    email: nva@unit.no
+  license:
+    name: MIT/NLOD 2.0
+    url: https://link.to.license
+  version: 1.0.0
+externalDocs:
+  description: Find out more about NVA
+  url: https://nva.unit.no
+servers:
+  - url: https://api.nva.unit.no/project
+paths:
+  /:
+    get:
+      x-amazon-apigateway-integration:
+        uri:
+          Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${NvaCristinProjectsFunction.Arn}/invocations
+        httpMethod: POST
+        type: "AWS_PROXY"
+      tags:
+        - Project
+      summary: List projects matching query
+      description: Returns a list of projects based on query parameters
+      operationId: ListProjects
+      parameters:
+        - name: title
+          in: query
+          description: Title search string
+          required: true
+          schema:
+            type: string
+            pattern: '.+'
+          example: 'Helse'
+        - name: language
+          in: query
+          description: Preferred language for some fields
+          required: false
+          schema:
+            type: string
+            pattern: '^[a-z]{2}$'
+            default: 'nb'
+          example: 'nb'
+        - name: page
+          in: query
+          description: Current page requested
+          required: false
+          schema:
+            type: string
+            pattern: '^[0-9]+$'
+            default: '1'
+          example: '3'
+        - name: results
+          in: query
+          description: Results per page
+          required: false
+          schema:
+            type: string
+            pattern: '^[0-9]+$'
+            default: '5'
+          example: '10'
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Projects'
+              examples:
+                objectExample:
+                  $ref: '#/components/examples/ProjectsExample'
+            application/ld+json:
+              schema:
+                $ref: '#/components/schemas/Projects'
+              examples:
+                objectExample:
+                  $ref: '#/components/examples/ProjectsExample'
+        400:
+          $ref: '#/components/responses/400'
+        500:
+          $ref: '#/components/responses/500'
+        502:
+          $ref: '#/components/responses/502'
+  /{id}:
+    get:
+      x-amazon-apigateway-integration:
+        uri:
+          Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${NvaCristinOneProjectFunction.Arn}/invocations
+        httpMethod: POST
+        type: "AWS_PROXY"
+      tags:
+        - Project
+      summary: Find project by ID
+      description: Returns a single project matching id parameter
+      operationId: GetProjectById
+      parameters:
```

Comment:
Does this have a query param for language? If yes, extract a Schema component for the language param and use that